crates/sncast/tests/e2e/script/deploy.rs

```diff
@@ -20,8 +20,8 @@ async fn test_with_calldata() {
         .current_dir(SCRIPTS_DIR.to_owned() + "/deploy")
         .args(args);
     snapbox.assert().success().stdout_matches(indoc! {r"
-        ...
-        command: script
-        status: success
+        [..]Compiling[..]
+        [..]Finished[..]
+        [..]Transaction hash[..]
```

Comment:
I'm not quite sure what change could have caused this tbh, perhaps I'm just missing something. do you mind pointing me to the exact place you have in mind? 

My understanding of your change at the moment is that it should not change the commands output, so I guess this could be reverted and the test should work as it did previously

Reply:
Eh, I had the wrong contract class_hash in this test. Fixed and reverted the previous stdout_match. This response looks so correct! Thx for the catch!